Lab-2.ipynb

```diff
@@ -0,0 +1,623 @@
+      "text/plain": [
+       "<Figure size 2016x504 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "p4 = convolve(\"Poisson\",4,Xbkg)\n",
+    "plotter(\"Poisson\",p4,4,\"avg\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "These graphs show how a Poisson distribution changes when we use convolution for averaging. To do this, we use the convolution operator again to sum, then divide to keep the mean constant at X, which we initially chose to be X = 2 mean cosmic ray counts. For simplicity, I had my convolution function use x values from 0 to 9 (10 total), so to simulate averaging, my plotter function similarly restricts the x-axis to (0,9) and hold the mean constant. The x values corresponding to the probabilities calculated using the convolution function are also restricted to this range.\n",
+    "\n",
+    "When averaging the days, the Poisson probability mass
```

Comment:
2.A: The width is altered as well? How so. Elaborate. Other than that looks good!